AdSpecter-iOS/Unity/ASRUnityAdLoader.m

```diff
@@ -0,0 +1,54 @@
+//
+//  ASRUnityAdLoader.m
+//  AdSpecter-iOS
+//
+//  Created by Adam Proschek on 3/20/18.
+//  Copyright © 2018 John Li. All rights reserved.
+//
+
+#import "ASRUnityAdLoader.h"
+#import <AdSpecter_iOS/AdSpecter_iOS-Swift.h>
+
+@interface ASRUnityAdLoader() <ASRAdLoaderDelegate>
+
+@property (nonatomic, strong) ASRAdLoader *adLoader;
+
+@end
+
+@implementation ASRUnityAdLoader
+
+- (instancetype)initWithImageLoadHandler:(ASRAdLoaderImageLoadedHandler)imageLoadedHandler {
```

Comment:
try to assign after object is initialized

currently passed into initWithImageLoadHandler. write init without it